cypress/e2e/04-covid_only_tests.cy.js

```diff
@@ -116,7 +116,10 @@ describe("Conducting a COVID test from:", () => {
         cy.contains("label", "Yes").click();
       });
 
-    cy.contains("legend", "Is the patient currently experiencing or showing signs of symptoms?")
+    cy.contains(
```

Comment:
⚠️ After running `yarn lint:write` locally this file was updated following `prettier` update